### ISSUE TITLE
BinaryDataNotification.data_var_added bug

### ISSUE THREAD

**Issue by hanssl** [Vector35/binaryninja-api#2020]
Binary Ninja Version: 2.1.2263, 4d2bf25a
Platform: Ubuntu 20.04.1 LTS

The var parameter of the 'def data_var_added(self, view, var):' callback method does not allow access to (code_refs, data_refs, data_refs_from).  If I access any of these fields in the notification callback then binarynina throws an error such as: 

'AttributeError: 'LP_BNBinaryView' object has no attribute 'get_code_refs'

It may be a race condition with that DataVariable and the view because if I first query the view for the same var (e.g. my_var = view.data_vars[var.address]) then I can access the (code_refs, data_refs, data_refs_from) fields of my_var without issue.

**Comment by plafosse** [comment]
Looks like it is returning the raw handle to the BinaryView object. You can convert this to a python BinaryView object as a workaround but that code would likely need to change when we issue a fix for this.

**Comment by negasora** [comment]
I'm not able to reproduce this on 2.1.2466 with this code:

```python
from binaryninja import BinaryDataNotification, open_view

class mynotifs(BinaryDataNotification):
    def data_var_added(self, view, var):
        print(view.get_code_refs(0x1000))
        print(view.get_data_refs(0x1000))
        print(view.get_data_refs_from(0x1000))

with open_view('/bin/true') as bv:
    bv.register_notification(mynotifs())
    bv.define_user_data_var(0x6323, bv.parse_type_string('int asd')[0])
```

**Comment by negasora** [comment]
Fixed in 2.2.2501-dev
